packages/plugins/other/visitor-plugin-common/src/selection-set-to-object.ts

```diff
@@ -512,4 +515,8 @@ export class SelectionSetToObject<Config extends ParsedDocumentsConfig = ParsedD
       suffix: typeName ? `_${typeName}_${suffix}` : suffix,
     });
   }
+
+  protected resolveDirectives(directives: any): boolean {
+    return directives.length > 0;
```

Comment:
Here we need to check the actual directives, right?

Reply:
sure. not implemented yet.